packages/demo/ozone-components-demo/elements/demo/demo-header/demo-header.html

```diff
@@ -109,7 +111,7 @@
         <paper-tab class="header-btn">Documentation</paper-tab>
         </paper-tabs>
         <div>Ozone-components Version: {{version}}</div>
-        <ozone-logout></ozone-logout>
+        <ozone-logout> <paper-icon-button  slot="logout-div" id="logoutBtn" icon="power-settings-new"></paper-icon-button></ozone-logout>
```

Comment:
Where is ozone-logout imported ?

Reply:
it's imported with the package ozone-login-form.
As it's a very small package I did not split it.